src/components/Table/Table.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import axios from 'axios';
+
+export default function InventoryTable() {
+    const inventory = [];
+    const [data, setData] = useState([]);
+    const [loading, setLoading] = useState(false);
+    const [error, setError] = useState(null);
+    const columns = [
+        { name: 'type', label: 'Resource Type' },
+        { name: 'fault', label: 'Fault State' },
+        { name: 'state', label: 'Status' },
+        { name: 'createdTime', label: 'Time Created' },
+        { name: 'lease', label: 'Lease Availability' },
+
+    ];
+    const rows = [
+        { type: 'Datacenter', status: { fault: '', state: '', createdTime: '', lease: '' } },
+    ];
+
+    const getData = async () => {
+        try {
+            const data = await axios.get(
+                'http://localhost:9999/api/v1/resources'
```

Comment:
all the hardcode pointing to the local host 9999, 8080 you can either put in an environment file, or specify in webpack/proxy configuration file, so you can switch to different servers more easily, and you will need that change that for ui deployment later, when the api servers are not running in localhost